orange_ecom_starter.theme

```diff
@@ -548,3 +556,10 @@ function orange_ecom_starter_preprocess_swiftmailer(&$variables) {
 function orange_ecom_starter_preprocess_commerce_cart_block(&$variables) {
   $variables['attributes']['class'][] = 'cart--cart-block';
 }
+
+/**
+ * Implements hook_commerce_checkout_pane_info_alter(&$checkout_panes).
+ */
+function orange_ecom_starter_commerce_checkout_pane_info_alter(&$checkout_panes) {
+  if($checkout_panes['shipping_information']) { return false; }
```

Comment:
What's the purpose of this? I don't think it has any effect unless you change something in `$checkout_panes`.

There's two code standards issues:
- `if(` should be `if (`.
- The body of this block and the `}` should each be on new lines.

If this *does* do something, please add a comment for what it does and why we need to do it that way - I'm sure I wouldn't be the only one confused by it.

Reply:
This has been removed completely since it didn't actually do anything.